modules/tinymce/CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Certain HTML content when inserted could cause the editor to crash #TINY-7756
 - Unbinding a native event handler inside the `remove` event caused an exception that blocked editor removal #TINY-7730
 - Disabled nested menu items could still be opened #TINY-7700
+- `Imagetools` buttons were enabled for remote images without `imagetools_proxy` set #TINY-7772
```

Comment:
Should we have a entry in `imagetools` changelog?

Reply:
It doesn't look like anything has changed in the `imagetools` module, you've only edited things in the plugin. So I'd say no

Reply:
```suggestion
- `imagetools` buttons were incorrectly enabled for remote images without `imagetools_proxy` set #TINY-7772
```
If we're using backticks then we need to use the actual name.